pkg/config/config.go

```diff
@@ -43,6 +43,7 @@ var (
 				DeployRunInit:           true,
 				InitRunReconfigure:      true,
 				AutoGenerateBackendFile: true,
+				AppendUserAgent:         "Cloud Posse Atmos v1.181 (https://atmos.tools)",
```

Comment:
Why are we hardcoding the atmos version?


```suggestion
				AppendUserAgent:         "Cloud Posse Atmos v1.181 (https://atmos.tools)",
```

Reply:
Also, please change the format to:

`Atmos/1.181 (Cloud Posse; +https://atmos.tools)`

Reply:
resolved